CONTRIBUTING.md

```diff
@@ -7,24 +7,32 @@ Getting started
 If you want to help but don't know where to start, here are some
 low-risk/isolated tasks:
 
-- [Merge a Vim patch].
 - Try a [good first issue](../../labels/good-first-issue) or [complexity:low] issue.
 - Fix bugs found by [Clang](#clang-scan-build), [PVS](#pvs-studio) or
   [Coverity](#coverity).
 - [Improve documentation][wiki-contribute-help]
+- [Merge a Vim patch]. [Runtime updates](https://github.com/vim/vim/search?q=update+runtime+files&type=commits) are easiest to get started.
 
 Reporting problems
 ------------------
 
+Before opening a new issue:
+
 - [Check the FAQ][wiki-faq].
 - [Search existing issues][github-issues] (including closed!)
 - Update Neovim to the latest version to see if your problem persists.
-- [Bisect](https://neovim.io/doc/user/starting.html#bisect) your config: disable plugins incrementally, to narrow down the cause of the issue.
-- [Bisect][git-bisect] Neovim's source code to find the cause of a regression, if you can. This is _extremely_ helpful.
+
+When reporting an issue:
+
+- First try to reproduce with `nvim --clean` ("factory defaults").
+- [Bisect](https://neovim.io/doc/user/starting.html#bisect) your config: disable plugins and remove settings incrementally, to narrow down the cause of the issue.
+- If the problem involves plugins, preparing a [minimal config](https://github.com/neovim/neovim/blob/master/contrib/minimal.lua) that reproduces the problem with `nvim -u minimal.lua` can be very helpful.
+- Check the logs: `:edit $NVIM_LOG_FILE`.
+- For shell-related problems: try `env -i TERM=ansi-256color "$(which nvim)"`.
+- Include `cmake --system-information` for build-related issues.
+- If there issue is a regression (i.e., the problem did not occur in an earlier version), [bisect][git-bisect] Neovim's source code to find exactly when it was introduced. This is _extremely_ helpful.
```

Comment:
typos. also the new, more wording is more verbose without really gaining anything